[Term Entry] SwiftUI Views: Circle

girijakar opened the pull request:
### Description

New term entry for SwiftUI Circle. Added a new folder, new image, and new file for issue #1092. Also, added a new tag in the tags.md
Closes #1092 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

SSwiniarski commented on content/swiftui/concepts/views/terms/circle/circle.md:
```diff
@@ -0,0 +1,45 @@
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Circle'
```
"Circle" may not be a broad enough topic for a tag.

SSwiniarski commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,45 @@
+//Properties, such as fill, frame, borders etc.
+```
+
+The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirement.
````
```suggestion
The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
```

SSwiniarski commented on documentation/tags.md:
```diff
@@ -46,6 +46,7 @@ Buttons
 Catch
 Characters
 Chatbots
+Circle
```
"Circle" may not be broad enough for a tag.

girijakar commented:
@SSwiniarski, I implemented your changes.
I have updated the tag from "Circle" to "SwiftUI Circles" in both the circle.md and tags.md. Let me know if this is fine, otherwise, I'll remove this tag.

SSwiniarski commented on content/swiftui/concepts/views/terms/circle/circle.md:
```diff
@@ -0,0 +1,45 @@
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'SwiftUI Circles'
```
```suggestion
```

SSwiniarski commented on documentation/tags.md:
```diff
@@ -256,6 +256,7 @@ Structure
 Style
 SwiftUI
 SwiftUI Views
+SwiftUI Circles
```
I'd get rid of this. The subject is too narrow to be a tag.

SSwiniarski reviewed (changes requested):
@girijakar I had some more comments, basically to remove the SwiftUI Circle tag as it's way too narrow in focus.

girijakar commented:
@SSwiniarski, removed the tag as suggested.

SSwiniarski reviewed (approved):
@girijakar Looks good for second review

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Circle'
+Description: 'Pre-built shape which can be colored and positioned as desired.'
```
```suggestion
Description: 'Pre-built circular shape which can be colored and positioned as desired.'
```

yangc95 replied:
Instead if saying "colored and positioned" can we say "visually modified?" This makes it broad as to what we can change about the circle view.

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
```diff
@@ -0,0 +1,44 @@
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules, which can be generated, coloured, and positioned as desired.
```
```suggestion
SwiftUI includes various pre-built shapes, like rectangles, **circles**, and capsules which can be generated, coloured, and positioned as desired.
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+
+```pseudo
+Circle()
+//Properties, such as fill, frame, borders etc.
````
```suggestion
// Modifiers, such as fill, frame, borders etc.
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+//Properties, such as fill, frame, borders etc.
+```
+
+The `"Circle"` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
````
```suggestion
`Circle` is one of the pre-built shapes. The user can use different properties such as fill, stroke, alignment, size to adjust the circle according to their requirements.
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
```diff
@@ -0,0 +1,44 @@
+
+## Example
+
+The following example make a `"Circle"` which is filled with purple color:
```
```suggestion
The following example make a `Circle` that is purple:
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+  }
+```
+
+The above code generates a circle having dimensions of 200x200.
````
Can we explicitly define the measurement unit

girijakar replied:
Yes, I added the unit.

yangc95 reviewed (commented):
@girijakar I think this is very well-written! Please tag me once you take a look over my comments.

Thanks!

girijakar commented:
@yangc95, implemented all your changes. Please, have a look and let me know if any more changes are required.

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+
+```pseudo
+Circle()
+// Modifiers, such as fill, frame, and borders
````
```suggestion
// Modifiers
```
 Let's keep the syntax code snippet like this and explain in the paragraph below

yangc95 replied:
It'd also be great if we can include that key term below

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+// Modifiers, such as fill, frame, and borders
+```
+
+`Circle` is one of the pre-built shapes. A developer can use different properties such as fill, stroke, alignment, and size to adjust the circle according to their requirements.
````
```suggestion
`Circle` is one of the pre-built shapes. A developer can use different properties such as fill, stroke, alignment, and size to modify or adjust the circle according to their requirements.
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+```swift
+  var body: some View{
+    Circle()
+      .fill(.purple)
+      .frame(width: 200; height: 200)
````
```suggestion
    .fill(.purple)
    .frame(width: 200; height: 200)
```

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+  }
+```
+
+The above code generates a circle having dimensions of 200x200 points.
````
```suggestion
The above code generates a circle having dimensions of `200`x`200` points.
```

yangc95 reviewed (commented):
@girijakar thanks for making those changes! I have a few more comments that need to be addressed prior to merge.

girijakar commented:
@yangc95, implemented the latest changes. Please check :)

yangc95 commented on content/swiftui/concepts/views/terms/circle/circle.md:
````diff
@@ -0,0 +1,44 @@
+The following example make a `Circle` that is purple:
+
+```swift
+  var body: some View{
````
```suggestion
  var body: some View {
```

yangc95 commented:
@girijakar I made some changes prior to merge here:
[649912c](https://github.com/Codecademy/docs/pull/1138/commits/649912c4b0ead0a8fb47a0e27c540681cc1c0606) & [e13f059](https://github.com/Codecademy/docs/pull/1138/commits/e13f0590eed099a9e3a4f47b2c999a74f95048e1)

Live link: https://www.codecademy.com/resources/docs/swiftui/views/circle

cc: @SSwiniarski
